### PR TITLE
Update installing erlang

### PR DESCRIPTION
**Pull request by erjohnson**

### CODE REVIEW

**Comment by laurenrother** [comment]
While we're updating this document, let's give it a copy-edit and some reorganization, too.

For instance, the first paragraph can be altered like:
"~~While~~ Pre-packaged versions of Riak include an Erlang installation. If you are building Riak from source, you will need to install [Basho's patched version of Erlang](http://s3.amazonaws.com/downloads.basho.com/erlang/otp_src_R16B02-basho8.tar.gz) by hand. This page will walk you through how to install Erlang on your own. ~~We strongly recommend using Basho's patched version of Erlang to install Riak 2.0. All of the patches in this version have been incorporated into later versions of the official Erlang/OTP release.~~"

I'd like to see the prerequisites called out in the 3rd paragraph actually listed out under a "Prerequisites" heading, so people know what they need quickly/easily without reading a block of text.

Additionally, I'd like the user to be able to see their install options in a list so they don't need to scroll through the document for their OS or preferred method of install. Perhaps including a brief note about what each heading includes (For instance, who can use kerl and why would they choose to. Or, what methods are available as a Mac user: source, homebrew, macports.) 

I'd like to see the install instructions listed out in steps, where possible. And there are a host of prereqs for Mac OS X, FreeBSD, or Solaris (including prereqs for those OSes even if installing w/ kerl) that are buried mid-page. They need to be moved under the Prerequisites heading and then need to be referenced at the very beginning of each applicable section (So, before the Mac install sections start, a note stating "Make sure you've installed the prerequisites before you start this section" with a link to the heading.)

I really like the addition of the "confirm this was successful" part. :zap:

File: source/languages/en/riak/ops/building/installing/erlang.md [hunk 1]
```diff
@@ -121,7 +121,8 @@ installed the necessary dependencies.
 Use this command to install the required dependency packages:
 
 ```bash
-sudo apt-get install build-essential libncurses5-dev openssl libssl-dev fop xsltproc unixodbc-dev
+sudo apt-get update
+sudo apt-get -y install build-essential autoconf libpam0g-dev maven openjdk-7-jdk
```
Comment (laurenrother) [inline source/languages/en/riak/ops/building/installing/erlang.md]: Sorry, I think I led you astray a bit. I was guessing that the old way of installing was outdated, but wasn't 100% for sure that the way in the wiki I linked is the BEST way. 

So, for instance, I think maybe mixing the two is best? Perhaps maintaining the openssl and unixodbc dependencies. 

It's worth double-checking with someone in Engineering to get a thumbs-up on the recommended way (to make sure we capture all the dependencies, etc.).

File: source/languages/en/riak/ops/building/installing/erlang.md [hunk 2]
```diff
@@ -141,6 +142,21 @@ To install packages for graphics support, use this command:
...
+```
+
+Then run the following to update your `PATH`:
+
+```bash
+export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
+PATH=$JAVA_HOME/bin:$PATH
+PATH=~/bin:$PATH
+export PATH
+```
```
Comment (laurenrother) [inline source/languages/en/riak/ops/building/installing/erlang.md]: Likewise here, let's just double-check with someone in Eng to make sure that this is a solid recommendation for every user (and not just something that's best for internal devs).

**Comment by erjohnson** [comment]
RFC :speech_balloon:

New commit splits prerequisite & installation instructions per OS. Also added separate sections for kerl instructions.

Still working on dependency portions for FreeBSD/Solaris & RHEL/CentOS.

**Comment by laurenrother** [comment]
This is totally awesome! :clap: :tada:

**Comment by laurenrother** [comment]
:cake: :fish_cake:

File: source/languages/en/riak/ops/building/installing/erlang.md [hunk 3]
```diff
@@ -13,189 +14,405 @@ moved: {
...
+## Installing with kerl
+
+First make sure you have installed the necessary dependencies and prerequisites found in [[kerl Prerequisites|Installing Erlang#kerl-Prerequisites]].
+
+With [kerl](https://github.com/yrashk/kerl)  installed, you can install Basho's recommended version of
+Erlang [from Github](https://github.com/basho/otp) using the following
+command:
+
+```bash
+./kerl build git git://github.com/basho/otp.git OTP_R16B02_basho8 R16B02-basho8
```
Comment (lukebakken) [inline source/languages/en/riak/ops/building/installing/erlang.md]: Just FYI, we're up to [`OTP_R16B02_basho9`](https://github.com/basho/otp/tree/OTP_R16B02_basho9) now, though I'm not 100% sure if this is the version that ships with Riak.

Comment (laurenrother) [inline source/languages/en/riak/ops/building/installing/erlang.md]: @erjohnson Can you follow up on this plz?

**Pull request merged**